main.rb

```diff
@@ -59,31 +56,29 @@ def welcome_screen
   welcome = "*"*welcome.length + "\n" + welcome + "\n" + "*"*welcome.length
 end
 
-def setup_player(num_of_player, name, symbol)
+def setup_player(num_of_player, name)
   print "Please insert the name of the player #{num_of_player}: "
   player_name = gets.strip.capitalize
   player_name = player_name == "" ? name : player_name
-  print "Select your symbol[default: #{symbol}]: "
-  player_sym = gets.strip.upcase
-  player_sym = player_sym == "" ? symbol : player_sym
 
-  return player_name, player_sym
+  return player_name
 end
 
-def make_a_move(player, board)
+def setup_move(player, board)
   begin
     puts "Enter the value from 1 to #{board.size**2}"
     print "#{player.name} your turn: "
     next_move = gets.match(/\d+/)[0].to_i
     raise "Please enter a number in the range." unless (next_move).between?(1, board.size**2)
-    player.play(next_move)
-    raise "Please select an empty position." unless board.fill_table(player.value, player.symbol)
+    raise "Please select an empty position." unless board.read_cell(next_move)
   rescue NoMethodError
     puts "Please enter only numeric values"
     retry
   rescue RuntimeError => e
     puts "#{e}"
```

Comment:
Prefer `to_s` over string interpolation

Reply:
@heyapricot  In which line should we  put the to_s? if we place it in ` puts "Enter the value from 1 to #{board.size**2}"` the solution would be `puts "Enter the value from 1 to"+(board.size**2).to_s` 
but why not use an interpolation here ?

Reply:
Yes, this one is on me. I wasn't clear enough 🤦🏻‍♂️
When you need the result string to contain *ONLY* the content of a variable, use `to_s`
Otherwise, use string interpolation (E.g if you need to put the contents of a variable in a message string)

Is this clear?

Reply:
Ok got it and changed.